inception-search-mtas/src/main/java/de/tudarmstadt/ukp/inception/search/index/mtas/IndexingConstants.java

```diff
@@ -0,0 +1,16 @@
+package de.tudarmstadt.ukp.inception.search.index.mtas;
+
+public class IndexingConstants {
+
+    public static String KB = "kb";
+    public static String kbEntity = "KB.Entity";
+    public static String kbConcept = "KBConcept";
```

Comment:
Are these constants what ends up in the query string? In the paper, we suggested to changed them. Please update accordingly.

Reply:
They are already done. Here are they : `indexKBConcept` & `indexKBInstance`

Reply:
Well, then I guess `kbEntity`and `kbConcept` can be removed?

Reply:
Why are these not final?